examples/java7/build-java7.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+set -e
+
+which acbuild || (echo "Make sure acbuild is on your path"; exit 1)
+
+TMPDIR=$(mktemp -d)
+PACKAGESDIR=$(mktemp -d)
+
+rmDirectories() {
+    rm -rf ${TMPDIR}
+    rm -rf ${PACKAGESDIR}
+}
+trap rmDirectories EXIT
+
+pushd ${PACKAGESDIR}
+apt-get download \
```

Comment:
IMO this should go into separate .txt file which then could be used like `$(cat deb-requirements.txt)` - any further modification of this list would not touch this file.